Nodejs/Common/Telemetry/TelemetryEvents.cs

```diff
@@ -20,5 +20,7 @@ namespace Microsoft.NodejsTools.Telemetry {
     /// </summary>
     internal static class TelemetryEvents {
         public const string ProjectImported = "ProjectImported";
+        public const string IntelliSenseActivatedForProject = "IntelliSenseActivatedForProject";
```

Comment:
Yep AnalysisLevel would be better for consistency. For technical discussions, we typically make a distinction between "IntelliSense" - the VS-branded user experience - and the actual engine that's providing the code intelligence.